src/Umbraco.Web/Models/ContentEditing/ContentItemBasic.cs

```diff
@@ -34,6 +34,9 @@ public class ContentItemBasic : EntityBasic
         [DataMember(Name = "updater")]
         public UserBasic Updater { get; set; }
 
+        [DataMember(Name = "contentTypeId")]

```

Comment:
There is already `contentTypeAlias` on this class, there should be no need to also have the Id since the content type can be looked up by alias, this will avoid all of the extra mappings needed too.